pl_examples/run_examples.sh

```diff
@@ -11,6 +11,23 @@ args="
   --trainer.limit_predict_batches=2
 "
 
-python "${dir_path}/basic_examples/simple_image_classifier.py" ${args} "$@"
 python "${dir_path}/basic_examples/backbone_image_classifier.py" ${args} "$@"
 python "${dir_path}/basic_examples/autoencoder.py" ${args} "$@"
+
+
+args="
+  --trainer.max_epochs=1
+  --trainer.limit_train_batches=2
+  --trainer.limit_val_batches=2
+  --trainer.limit_test_batches=2
+  --trainer.limit_predict_batches=2
+"
+
+python "${dir_path}/basic_examples/mnist_examples/image_classifier_4_lightning.py" ${args} "$@"
```

Comment:
why this strange order 4-5-1-2-3?

Reply:
Because 4-5 is using PyTorch Lightning and therefore the arg above. The others won't

Reply:
we can switch the order back

Reply:
ok, so just let just add comment for our future us :]